### PR TITLE
Task-2371 uploader (text): usx and json not being uploaded

### PR DESCRIPTION
**Pull request by vichugofsl**
# Description
1. Removed incorrect logic that was preventing the upload of primary and derived filesets.

2. Additionally, I removed unnecessary logic that attempted to update the copyright information of an existing derived fileset using the same information.

# Task
[Bug 2371](https://dev.azure.com/keyholesoftware/FCBH%20-%20Bible%20Brain/_workitems/edit/2371): uploader (text): usx and json not being uploaded

# How to test

You can run the following command and the outcome should be the following:
`````shell
python3 load/DBPLoadController.py test s3://dbp-etl-upload-dev-ya1mbvdty8tlq15r/ "2024-11-26-14-27-01/Semai_N2 & P2 SEAWYI_USX"
`````

### Outcome
[log_upload_filesets.txt](https://github.com/user-attachments/files/17922269/log_upload_filesets.txt)

### CODE REVIEW

File: load/S3Utility.py [hunk 1]
```diff
@@ -48,10 +48,10 @@ def uploadAllFilesets(self, filesets):
 						print("uploadAllFilesets. outFileset NOT added to database list: %s" % (outFileset.csvFilename))
 			elif inp.typeCode == "text":
 				subTypeCode = inp.subTypeCode()
-				if subTypeCode in {"text_plain", "text_usx", "text_json"}:
-					InputFileset.database.append(inp) ## Temp until text_format is ready
+				if subTypeCode in {"text_plain"}:
```
Comment (vichugofsl) [inline load/S3Utility.py:51]: @bradflood, my bad, I changed it again because if the fileset is text_usx or text_json, the uploadFileset method will add the fileset to the InputFileset.database slice and upload the file. The issue was that it was not invoking the method uploadFileset for text_usx and text_json.

Comment (bradflood) [reply load/S3Utility.py:51]: what was the reason you changed it initially? You just undid it, so I would like to know the effect on this on the original issue you were addressing

File: load/UpdateDBPLicensorTables.py [hunk 2]
```diff
@@ -52,18 +52,13 @@ def processFileset(self, primaryFilesetId, derivedHashId):
 
 		for (copyright, copyrightDate, copyrightDescription) in copyrights:
 			hashIdExisting = self.db.selectScalar("\
-				SELECT bible_fileset_copyrights.copyright,\
-					bible_fileset_copyrights.copyright_date,\
-					bible_fileset_copyrights.copyright_description,\
-					bible_fileset_copyrights.open_access\
+				SELECT bible_fileset_copyrights.copyright\
```
Comment (bradflood) [inline load/UpdateDBPLicensorTables.py:55]: this seems unrelated. 
and it removes copyright_date. 

I think we need to be more focused, so that the PR description correctly represents the code being changed.

Does copyright_date need to be added back in?

**Review by bradflood — CHANGES_REQUESTED** [review]

File: load/UpdateDBPLicensorTables.py (hunk 2 above)
Comment (vichugofsl) [reply load/UpdateDBPLicensorTables.py:55]: @bradflood, I made this change because, while investigating the issue (task-2371), I noticed that the ETL process was always creating a SQL statement to update the copyright of a derived fileset using the same existing information. As a result, I removed this no longer necessary line: [link](https://github.com/faithcomesbyhearing/dbp-etl/pull/170/files#diff-5bd192f19936aa1de297001dae83cd0ca789f285243182525ab307410295613dL65).

Regarding the query, it won't remove the copyright_date. This query is used to check whether the derived fileset has copyright content. The copyright_date value is retrieved from the primary fileset in the following line: [link](https://github.com/faithcomesbyhearing/dbp-etl/blob/75bf8788e8a6c41068d341f69588df6c0fbdc11f/load/UpdateDBPLicensorTables.py#L51).
